AlphaWallet.xcodeproj/project.pbxproj

```diff
@@ -6052,6 +6053,13 @@
 			path = Coordinators;
 			sourceTree = "<group>";
 		};
+		E7F44F672852BF69006A7C93 /* New Group */ = {
```

Comment:
Would you help remove this and the other references to "New Group" that was added to the project file?

Reply:
Yes